server/routes/link/VerifyLinkController.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express'
+import MagicLinkService from '../../services/link/MagicLinkService'
+import logger from '../../../logger'
+
+export default class VerifyLinkController {
+  constructor(private readonly magicLinkService: MagicLinkService) {}
+
+  async verifyLink(req: Request, res: Response): Promise<void> {
+    const secret = req.query.secret as string
+    if (!secret) {
+      logger.warn('User attempted to verify a link without a secret on the query string')
```

Comment:
I'm not sure what we're going to do with this log message - it might just become noise. I think it would be good at the end of this Epic to break out another ticket about metrics to capture around the login journey. It might be good to be able to see how many obviously fraudulent attempts to sign in vs genuinue sign ins for example.

Reply:
OK, I'll remove it for now